index.html

```diff
@@ -54,119 +62,123 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
+            </div>
+            <p class="card-body">
+              Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+            </p>
+            <ul class="tech-stack facebook">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items rails" type="button">ruby on rails</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li>
+                <button class="tech-items" type="button">javascript</button>
+              </li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
           </div>
-          <p class="card-body">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="tech-stack">
-            <li><button class="tech-items" type="button">html</button></li>
-            <li><button class="tech-items" type="button">css</button></li>
-            <li><button class="tech-items" type="button">javascript</button></li>
-          </ul>
-          <a class="card-button" href="#">See Project</a>
         </div>
       </div>
-      
+
       <div class="card">
         <div class="card-element">
-          <div class="img-container">
-            <img class="work-image" src="./Images/Tonic-two.png" alt="work-Image" />
-          </div>
-          <h2 class="card-title">Tonic</h2>
-          <div class="card-section">
-            <p class="para-one">CANOPY</p>
-            <i class="fa fa-circle"></i>
-            <p class="para-two">Back End Dev</p>
-            <i class="fa fa-circle"></i>
-            <p class="para-two">2015</p>
+          <img class="work-image" src="./Images/Tonic-two.png" alt="work-Image" />
+          <div class="card-content-container">
+            <h2 class="card-title">Facebook 360</h2>
+            <div class="card-section">
+              <p class="para-one">FACEBOOK</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">Full Stack Dev</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">2015</p>
+            </div>
+            <p class="card-body">
+              Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+            </p>
+            <ul class="tech-stack facebook">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items rails" type="button">ruby on rails</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li>
+                <button class="tech-items" type="button">javascript</button>
+              </li>
+            </ul>
+            <a class="card-button blue-button" href="#">See Project</a>
```

Comment:
- [ ] This button doesn't need to be blue. In the template, it was simply used to show the interaction on hover, it is just like the other buttons 👍

Reply:
Thanks for the remark. I will fix these.